conda/recipes/libraft/meta.yaml

```diff
@@ -42,23 +41,35 @@ outputs:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('cuda') }} {{ cuda_version }}
-        - sysroot_{{ target_platform }} {{ sysroot_version }}
         - cmake {{ cmake_version }}
+        - ninja
+        - sysroot_{{ target_platform }} {{ sysroot_version }}
       host:
         - cudatoolkit {{ cuda_version }}.*
+        - cuda-profiler-api {{ cuda_profiler_api_version }}
```

Comment:
should `cuda-profiler-api` be in `dependencies.yaml`? I don't see it there currently.

Reply:
Yes. Treat `cuda-profiler-api` and `cuda-nvtx` and similar packages the same way as `libcu*`.

Resolved in 406212d27816ca0e5048949d68ff78a38f9a3943.